src/patternfly/components/Popover/popover.scss

```diff
@@ -215,9 +208,12 @@
   box-shadow: var(--pf-c-popover__arrow--BoxShadow);
 }
 
+.pf-c-popover__header {
+  margin-bottom: var(--pf-c-popover__header--MarginBottom);
+}
+
 .pf-c-popover__title {
   flex: 0 0 auto;
-  margin-bottom: var(--pf-c-popover__title--MarginBottom);
   font-family: var(--pf-c-popover__title--FontFamily);
   font-size: var(--pf-c-popover__title--FontSize);
   line-height: var(--pf-c-popover__title--LineHeight);
```

Comment:
Are we updating this the same way as empty state to move the title text styling to the title-text element (that can be replaced with a `<Title>` or whatever to opt-out of our styling)? 

There is also a `.pf-m-icon` class modifier on `__title` that creates a flex layout. Since the title text will be in `title-text` now, I think we can just make this element a flex layout by default.